core/types/transaction.go

```diff
@@ -778,6 +778,11 @@ func (tx *Transaction) ConfirmationCtx(nodeLocation common.Location) int {
 	return ctx
 }
 
+// Conversion returns true if the transaction is a Quai to Qi or Qi to Quai conversion transaction
+func (tx *Transaction) Conversion() bool {
+	return tx.Type() == ExternalTxType && tx.ETXSender().Location().Equal(*tx.To().Location())
```

Comment:
isnt a better check to see if this transactions is crossing the ledger? Just checking if the to and the sender location just tells you if the transaction was to the current chain? stricter check would be the ledger check? this also works given the guarantee of etx type, but just a note